### DISCUSSION TITLE
How to save pinned images?

### DISCUSSION THREAD

**Discussion by ms-2525** [flameshot-org/flameshot#1967] [Q&A] [answered]
Is there any option to save the pinned images ??
There are no buttons on the pinned image and only thing you can do is double click on it to close the pinned image.

**Comment by mmahmoudian** [comment, marked as answer]
correct, at the moment there is no way to do that, but in  #954 there is a discussion to improve the pin tool. You can follow the discussion there and also express your opinion.
